### DISCUSSION TITLE
Bug: Can't add more then one extension

### DISCUSSION THREAD

**Discussion by Vrackko** [seleniumbase/SeleniumBase#2698] [Q&A] [answered]
I already asked this one (https://github.com/seleniumbase/SeleniumBase/discussions/2672) but i ran at the bug which i cant solve or find anything about that. 

So when i want to use one extension, which is stored in path: C:\Users\HP\Desktop\No Com\extensions\fpd
and with only one extension it works
`
driver = Driver(uc=True, agent=user_agent, proxy=proxy_cred, extension_dir='extensions/fpd', uc_cdp_events=True, user_data_dir=custom_user_data_dir)
`

Same thing with the second extension, which is in the path: C:\Users\HP\Desktop\No Com\extensions\httphm
`
driver = Driver(uc=True, agent=user_agent, proxy=proxy_cred, extension_dir='extensions/httphm', uc_cdp_events=True, user_data_dir=custom_user_data_dir)
`

So the plugins work, but when i combine them like this:
`
driver = Driver(uc=True, agent=user_agent, proxy=proxy_cred, extension_dir='extensions/httphm,extensions/fpd', uc_cdp_events=True, user_data_dir=custom_user_data_dir)
`

Only the first one loads properly, for the second one i get error:
![image](https://github.com/seleniumbase/SeleniumBase/assets/145341123/d141f403-bea8-418b-ae8c-505f3ecdd6fd)

Same thing if i swap them. Tried many different things but i always get that error pop up which looks at the different folder for the extension.

So the plugins arent the problem but pathing to the second one.

I found the path from the error pop up and saved my extension there: C:\Program Files\Google\Chrome\Application\123.0.6312.107\httphm

With this code it works:
`
driver = Driver(uc=True, agent=user_agent, proxy=proxy_cred, extension_dir='extensions/fpd,httphm', uc_cdp_events=True, user_data_dir=custom_user_data_dir)
`

But this is not my solution because i will need to push my script to the server. Is this a bug? I cant find anyone having this problem. Thanks for any kind of help.

**Comment by mdmintz** [comment, marked as answer]
Worked fine for me when I loaded three extension directories at the same time:

```bash
pytest basic_test.py --extension_dir="ad_block,sbase_ext,disable_csp" --uc --ftrace
```

<img width="750" alt="Screenshot 2024-04-15 at 11 30 19 AM" src="https://github.com/seleniumbase/SeleniumBase/assets/6788579/5a1fe148-ce80-49e7-8b7e-61dc7126fad1">

Your error shows a problem with your extension, which means it's on you to fix it. If it's not an issue with your extension, then there's an issue with your `user_data_dir`. Maybe try without setting a `user_data_dir` if that's the case.

**Comment by Vrackko** [comment]
Hi, i was on pause on this project because of school. Can you share example of code you used. Kind regards.

**Reply by mdmintz** [reply]
https://github.com/seleniumbase/SeleniumBase/discussions/2698#discussioncomment-9120110

**Reply by Vrackko** [reply]
I meant basic_test.py
This one?
https://github.com/seleniumbase/SeleniumBase/blob/419cedc8fba6bfce71add3ec915a92d67698c9a1/examples/basic_test.py#L4

**Reply by mdmintz** [reply]
That's the test. The command-line options are separate.

**Reply by Vrackko** [reply]
Hey, i tested it with basic_test.py and your pytest command. It worked

Then i tested my script with no changes and it worked perfectly. Don't know what the problem was but i am thankful for your help.
